modules/machineset-yaml-rhv.adoc

```diff
@@ -113,16 +115,20 @@ If you are using a version earlier than {rh-virtualization} 4.4.8, see link:http
 
 <16> Optional: Specify the size of the bootable disk in GiB.
 
-<17> Optional: List of the network interfaces of the VM. If you include this parameter, {product-title} discards all network interfaces from the template and creates new ones.
+<17> Optional: Specify the UUID of the storage domain for the compute node's disks. If none is provided, the compute node is created on the same storage domain as the control nodes. (default)
```

Comment:
as the ovirt template

Reply:
@Gal-Zaidman 

I don't think we need to say this here. There's a note below that says:

[NOTE]
====
Because {rh-virtualization} uses a template when creating a VM, if you do not specify a value for an optional parameter, {rh-virtualization} uses the value for that parameter that is specified in the template.
====

If we say it here, then we have to say it everywhere.